app/scripts/modules/core/src/application/config/applicationConfig.view.html

```diff
@@ -1,4 +1,4 @@
-<div class="container" sticky-headers>
+<div class="container" style="overflow-y: auto" sticky-headers>
```

Comment:
Added this style because the additional container divs are mucking with the scroll behavior.